### PR TITLE
Fix PV inverter error display and clean up product IDs

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

File: src/productinfo.h [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+#include <QQmlEngine>
+#include <QObject>
+
+namespace Victron {
+namespace VenusOS {
+
+class ProductInfo : public QObject
+{
+	Q_OBJECT
+	QML_NAMED_ELEMENT(ProductInfo)
```
Comment (DanielMcInnes) [inline src/productinfo.h:18]: why not just QML_ELEMENT?

File: src/productinfo.h [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	QML_SINGLETON
+
+public:
+	explicit ProductInfo(QObject * = nullptr) {}
+	~ProductInfo() override {}
+
+	enum ProductId_Alternator {
+		ProductId_Wakespeed = 0xB080, // VE_PROD_ID_WAKESPEED_WS500
```
Comment (DanielMcInnes) [inline src/productinfo.h:26]: Maybe prefix the enum entry with the enum title, eg. ProductId_Alternator_Wakespeed ?

File: src/productinfo.h [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+		ProductId_FischerPanda = 0xB040,
+		ProductId_ComAp = 0xB044,
+		ProductId_Dse = 0xB046,
+		ProductId_Cre = 0xB048,
+		ProductId_Deif = 0xB049,
+	};
+	Q_ENUM(ProductId_Genset)
+
+	Q_INVOKABLE bool isGensetProduct(int productId) {
+		static const QVector<int> productIds = {
```
Comment (DanielMcInnes) [inline src/productinfo.h:69]: maybe a switch/case instead of a QVector with static lifetime?

**Review by DanielMcInnes — APPROVED** [review]
lgtm, nits

File: src/productinfo.h (hunk 1 above)
Comment (blammit) [reply src/productinfo.h]: Oh true, the QML_NAMED_ELEMENT wasn't necessary. Fixed.

File: src/productinfo.h (hunk 2 above)
Comment (blammit) [reply src/productinfo.h]: Done, some of the others didn't have a clear grouping, so I added `Product_Misc` and put those enums there.

File: src/productinfo.h (hunk 3 above)
Comment (blammit) [reply src/productinfo.h]: Yes, much nicer. Fixed, thanks.

**Pull request merged**